devicon.json

```
@@ -3752,6 +3752,22 @@
         },
         "color": "#D1A41A",
         "aliases": []
+    },
+	{
+        "name": "tortoisegit",
+        "tags": [
+            "git"
+        ],
+        "versions": {
+            "svg": [
+                "original",
```

Just need to remove the `,` here. JSON is a bit strict on comma compared to normal JS objects.
```suggestion
                "original"
```